_includes/_code/lakes-interactive/lib/border-star-button.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:flutter/material.dart';
+// Uncomment lines 3 and 6 to view the visual layout at runtime.
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+class FavoriteWidget extends StatefulWidget {
+  @override
+  FavoriteWidgetState createState() => new FavoriteWidgetState();
+}
+
+class FavoriteWidgetState extends State<FavoriteWidget> {
```

Comment:
Supporting flutter-specific analysis has definitely been discussed.  See also https://github.com/dart-lang/sdk/issues/25736.  I'll add a note to myself to ask @devoncarew next time we chat if there are updates re: flutter writing its own analysis checks/lints, etc.